Add breadcrumbs activity

droberts-sea opened the pull request.

Commit:
Add breadcrumbs activity

Commit:
Merge remote-tracking branch 'origin/master' into dpr/breadcrumbs

CheezItMan commented on README.md:
````diff
@@ -138,12 +138,23 @@ Cut out the squares and put the segments of code in order in which they are ran,
+```
+
+Where `#` is the order in which the log line appears when the app is started. If the function shouldn't be called when the page is first loaded, replace `#` with `does not print`.
+
````
I would suggest an example:

So for the first function called in the application you should place:  `console.log(">>> BREADCRUMBS: 1");`

kariabancroft commented on README.md:
```diff
@@ -138,12 +138,23 @@ Cut out the squares and put the segments of code in order in which they are ran,
 ### Initial Page Load
-When the page initially loads, there is a lot of code in various parts of our application that are depended on to make sure it all comes together.
+**Activity:**
+When the page initially loads, there is a lot of code in various parts of our application that are depended on to make sure it all comes together. To help understand how this all fits together, let's leave ourselves a trail of breadcrumbs to follow.
```
Can you clarify the verbiage in this first sentence? It is sort of confusing

Commit:
Breadcrumbs Activity: Respond to comments in PR

Revise the first paragraph to be a little more clear
Provide an example of what a log line should look like

kariabancroft commented:
Updates look good!
